### PR TITLE
Tsm/translations

### PR DESCRIPTION
**Pull request by Caele**
Add use of translator. Strings are still subject to check from doc, but that will be done in the client - then updated here.
Also fix for dimensions limits.

### CODE REVIEW

File: src/locale/en-US.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Local translations when used outside of sense
+export default {
+  'Object.OrgChart': 'Org chart', // Name of chart, to be added to sense when native
+  'Object.OrgChart.Description':
+    'Displays the relations of an organization or similarly structured data. Starts at the top and allows for navigation through the hierachy.', // Description of chart, to be added to sense when native
+  'Object.OrgChart.MaxData': 'The maximum data limit is reached. The tree may display incorrectly.',
+  'Object.OrgChart.DummyRoot': 'Root root',
```
Comment (haxxmaxx) [inline src/locale/en-US.js]: why is this needed?

Comment (Caele) [reply src/locale/en-US.js]: good question, intended for tooltip on the extra root nodes, but probably not needed. Also strangely written, i'll remove it.

File: src/extension/meta.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-  "name": "Organizational Chart",
-  "description": "Chart to visualise organisations",
+  "name": "Org chart",
+  "description": "Displays the relations of an organization or similarly structured data. Starts at the top and allows for navigation through the hierachy.",
   "icon": "network",
   "preview": "preview.png",
   "author": "QlikTech International AB",
+  "supernova": true,
   "type": "visualization"
 }
```
Comment (niekvanstaveren) [inline src/extension/meta.json]: Would it be possible to have only meta.json or sn-org-chart.qext ?? The files are quite similar. Maybe something for after first release though

Comment (Caele) [reply src/extension/meta.json]: yea, the meta file is actually only for when building with `nebula sense`, that doesn't work right now and as it won't really help us anyway I haven't pursued that. We can mostly ignore that file for now, and likely remove it later.
